tests/test_requirements.py

```diff
@@ -73,6 +73,10 @@ def test_name_with_multiple_versions(self):
         req = Requirement("name>=3,<2")
         self._assert_requirement(req, "name", specifier="<2,>=3")
 
+    def test_name_with_multiple_versions_and_whitespace(self):
+        req = Requirement("name>=3, <2")
```

Comment:
what about also adding whitespace on the right side of the operator? imho it'd make sense to cover all relevant cases